lib/dolly/document.rb

```diff
@@ -51,7 +51,8 @@ def rev= value
       doc['_rev'] = value
     end
 
-    def save
+    def save options = {}
+      return false unless options[:validate] == false || valid?
```

Comment:
@seancookr this is consistent with activerecords implementation. Consistent behavior seems like a good use case and brings dolly a step closer to implementing validations. https://github.com/rails/rails/blob/master/activerecord/lib/active_record/validations.rb#L36

@rubenrails please include touch as an option as well. Crude example
```
if timestamps
  set_created_at
  set_updated_at unless options[:touch] == false
```